.github/workflows/main.yml

```diff
@@ -25,22 +25,20 @@ jobs:
           submodules: true
 
       - name: Setup Go environment
-          uses: actions/setup-go@v3.0.0
-          with:
-            # The Go version to download (if necessary) and use. Supports semver spec and ranges.
-            go-version: 1.18
-            # Set this option to true if you want the action to always check for the latest available version that satisfies the version spec
-            check-latest: true
+        uses: actions/setup-go@v3.0.0
+        with:
+          # The Go version to download (if necessary) and use. Supports semver spec and ranges.
+          go-version: 1.18
+          # Set this option to true if you want the action to always check for the latest available version that satisfies the version spec
+          check-latest: true
 
       # Setup tools
       - name: Setup tools
-        run: |
-          pip install pre-commit
-          pre-commit install
+        run: make
 
       # Check for pre-commit updates if it is a renovate PR
       - name: Pre-commit sweeper
-        run: module-assets/ci/pre-commit-sweeper.sh
+        run: common-dev-assets/module-assets/ci/pre-commit-sweeper.sh
```

Comment:
you just created a symlink for the `ci` folder, so I guess you can shorten this to use that: `ci/pre-commit-sweeper.sh`